<?xml version="1.0"?>
<launch>
    <!--param name="use_sim_time" value="true" /-->
    <node pkg="ndt_fuser" type="ndt_fuser_node" name="fuser_3d" output="screen">
	<param name="points_topic" value="/velodyne_points" />
	<param name="matchLaser" value="false" />
	<param name="size_x_meters" value="60" />
	<param name="size_y_meters" value="60" />
	<param name="size_z_meters" value="5" />
	<param name="resolution" value="0.4" />
	<param name="sensor_range" value="40." />
	
        <!-- Specific sensor offset parameters for the example file -->
	<param name="sensor_pose_x" value="0.8" />
	<param name="sensor_pose_z" value="1.03" />
	<param name="sensor_pose_t" value="0.0" />
	
	<param name="pose_init_x" value="0.0" />
	<param name="pose_init_y" value="0.0" />
	<param name="pose_init_t" value="0.0" />
	
	<param name="map_directory" value="$(find ndt_fuser)/maps/" />	
	<param name="map_name_prefix" value="ncfm_sim_03_gt" />
	
	<param name="beHMT" value="false" />
	<param name="match2D" value="true" />
	<param name="useOdometry" value="false" />
	<param name="odometry_topic" value="/robot1/odom" />
	
	<param name="do_soft_constraints" value="false" />
	<param name="initPoseFromTF" value="false" />
	<param name="tf_pose_frame" value="/robot1/base_footprint" />
	<param name="robot_frame" value="/robot1/base_footprint" />
	<param name="sensor_frame" value="/robot1/velodyne" />

        <param name="visualize" value="true" />
    </node>
</launch>
